configure.ac

```diff
@@ -321,6 +321,35 @@ if test ! x"$with_mandoc" = xno; then
 fi
 AM_CONDITIONAL([FVWM_BUILD_MANDOC], [test x"$with_mandoc" = xyes])
 
+AC_ARG_ENABLE(htmldoc,
+  AS_HELP_STRING([--enable-htmldoc],
+    [enable generation of html man pages]),
+  [ if test x"$enableval" = xyes; then
+		with_htmldoc="yes, check"
+	else
+		with_htmldoc="no"
+		problem_htmldoc=": Explicitly disabled"
+	fi ],
+	[ with_htmldoc="no" ]
+)
+
+if test ! x"$with_htmldoc" = xno; then
+	if test x"$ASCIIDOC" = x ; then
+		with_htmldoc="no"
+		problem_htmldoc=": No asciidoctor found in PATH"
+	elif test x"$SED" = x ; then
+		with_htmldoc="no"
+		problem_htmldoc=": No sed found in PATH"
+	elif test x"$HAVE_PERL_FOR_DOCS" = x0 ; then
```

Comment:
Why is this check necessary?

Reply:
I just copied the manpage checks, but the second commit removes these perl checks, since the POD is no longer being used.